docs/sql/index.md

```diff
@@ -1,407 +1,391 @@
+```ebnf
+literal = INTEGER | FLOAT | STRING | HEX | TRUE | FALSE ;
 ```
 
-Select all columns of all rows from the `inventory` table where the `inventory_id` is 2 or greater:
+The following describes how to construct literal values,
+for [SATS] data types, in Spacetime SQL.
 
-```sql
-SELECT * FROM inventory WHERE inventory_id > 1;
-```
+### Booleans
 
-### `INSERT`
+Booleans are represented using the canonical atoms `true` or `false`.
 
-An `INSERT INTO` statement inserts new rows into a table.
+### Integers
 
-One can insert one or more rows specified by value expressions.
+```ebnf
+INTEGER
+    = [ '+' | '-' ] NUM
+    | [ '+' | '-' ] NUM 'E' [ '+' | '-' ] NUM
+    ;
 
-The syntax of the `INSERT INTO` statement is:
+NUM
+    = DIGIT { DIGIT }
+    ;
 
-> **INSERT INTO** _table_name_ (_column_name_, ...) **VALUES** (_expr_, ...), ...;
-
-![sql-insert](/images/syntax/insert.svg)
+DIGIT
+    = 0..9
+    ;
+```
 
-#### Examples
+> NOTE: The concrete [SATS] type is inferred from the context.
 
-Insert a single row:
+### Floats
 
-```sql
-INSERT INTO inventory (inventory_id, name) VALUES (1, 'health1');
+```ebnf
+FLOAT
+    = [ '+' | '-' ] [ NUM ] '.' NUM
+    | [ '+' | '-' ] [ NUM ] '.' NUM 'E' [ '+' | '-' ] NUM
+    ;
 ```
 
-Insert two rows:
+> NOTE: The concrete [SATS] type is inferred from the context.
 
-```sql
-INSERT INTO inventory (inventory_id, name) VALUES (1, 'health1'), (2, 'health2');
-```
+### Strings
 
-### UPDATE
-
-An `UPDATE` statement changes the values of a set of specified columns in all rows of a table, optionally filtering the table to update only rows which satisfy a `WHERE` predicate.
+```ebnf
+STRING
+    = "'" { "''" | CHAR } "'"
+    ;
+```
 
-Columns not explicitly modified with the `SET` clause retain their previous values.
+`CHAR` is defined as a `utf-8` encoded unicode character.
 
-If the `WHERE` clause is absent, the effect is to update all rows in the table.
+### Hex
 
-The syntax of the `UPDATE` statement is
+```ebnf
+HEX
+    = 'X' "'" { HEXIT } "'"
+    | '0' 'x' { HEXIT }
+    ;
 
-> **UPDATE** _table_name_ **SET** > _column_name_ = _expr_, ...
-> {_WHERE expr_}?;
+HEXIT
+    = DIGIT | a..f | A..F
+    ;
+```
 
-![sql-update](/images/syntax/update.svg)
+Hex literals can represent [Identity], [Address], or binary types.
+The type is ultimately inferred from the context.
 
-#### Examples
+## Identifiers
 
-Set the `name` column of all rows from the `inventory` table with the `inventory_id` 1 to `'new name'`:
+```ebnf
+identifier
+    = LATIN { LATIN | DIGIT | '_' }
+    | '"' { '""' | CHAR } '"'
+    ;
 
-```sql
-UPDATE inventory
-  SET name = 'new name'
-  WHERE inventory_id = 1;
+LATIN
+    = a..z | A..Z
+    ;
 ```
 
-### DELETE
-
-A `DELETE` statement deletes rows that satisfy the `WHERE` clause from the specified table.
+Identifiers are tokens that identify database objects like tables or columns.
+Spacetime SQL supports both quoted and unquoted identifiers.
```

Comment:
What's the difference between a quoted and an unquoted identifier? When would I use one or the other?